### PR TITLE
RTL calendar fix - arrows fix and views fix

### PR DESCRIPTION
**Pull request by yosilevy**
## Proposed change
Fix for how calendar page supports RTL.

Before:

Note arrows are pointing wrong and view list has wrong borders:

image

After:

image

## Checklist

- [X ] The code change is tested and works locally.
- [X ] There is no commented out code in this PR.

### CODE REVIEW

File: src/panels/calendar/ha-full-calendar.ts [hunk 1]
```diff
@@ -124,26 +127,27 @@ export class HAFullCalendar extends LitElement {
...
                         .path=${mdiChevronLeft}
                         class="prev"
                         @click=${this._handlePrev}
                       >
-                      </ha-icon-button>
-                      <ha-icon-button
+                      </ha-icon-button-prev>
+                      <ha-icon-button-next
                         .label=${this.hass.localize("ui.common.next")}
                         .path=${mdiChevronRight}
```
Comment (zsarnett) [inline src/panels/calendar/ha-full-calendar.ts:139]: Not needed now

File: src/panels/calendar/ha-full-calendar.ts [hunk 2]
```diff
@@ -124,26 +127,27 @@ export class HAFullCalendar extends LitElement {
                           "ui.components.calendar.today"
                         )}</mwc-button
                       >
-                      <ha-icon-button
+                      <ha-icon-button-prev
                         .label=${this.hass.localize("ui.common.previous")}
                         .path=${mdiChevronLeft}
```
Comment (zsarnett) [inline src/panels/calendar/ha-full-calendar.ts:132]: same

File: src/panels/calendar/ha-full-calendar.ts (hunk 1 above)
Comment (yosilevy) [reply src/panels/calendar/ha-full-calendar.ts:139]: @zsarnett Why is it not needed? I created the -prev/-next components a few years back so they automatically adapt to RTL...

Comment (zsarnett) [reply src/panels/calendar/ha-full-calendar.ts:139]: The path isnt needed right?

Comment (yosilevy) [reply src/panels/calendar/ha-full-calendar.ts]: Oh right! Removed. The comment was on different lines so I didn't follow. Thanks.

**Comment by yosilevy** [comment]
@zsarnett I think I closed the threads so please review again. Sorry.

**Comment by yosilevy** [comment]
@zsarnett please review so we can merge. Thanks.

**Review by zsarnett — APPROVED** [review]

**Pull request merged**